src/screens/Swap/FormOrHistory/Form/index.js

```diff
@@ -134,7 +133,9 @@ const Form = ({
     <View style={styles.root}>
       <TrackScreen category="Swap" name="Form" />
       <Alert
+        id="transfer.swap.form.paraswapCta"
         type="hint"
+        closeable={true}
```

Comment:
remove the ={true}
Just `closeable` is shorthand usage that we should use for boolean.